workers/Dockerfile.tests

```diff
@@ -0,0 +1,3 @@
+FROM dr_worker
```

Comment:
Might be related to the CircleCI issue, if I'm following the Docker trail correctly.